README.md

```diff
@@ -224,7 +224,30 @@ The OpenTelemetry Operator *might* work on versions outside of the given range,
 | v0.29.0                | v1.19 to v1.21       |
 | v0.28.0                | v1.19 to v1.21       |
 | v0.27.0                | v1.19 to v1.21       |
-| v0.26.0                | v1.18 to v1.20       |
+
+
+### OpenTelemetry Operator vs. Cert Manager
+Since Cert-Manager deprecated *cert-manager.io/v1alpha2, cert-manager.io/v1alpha3, cert-manager.io/v1beta1, acme.cert-manager.io/v1alpha2, acme.cert-manager.io/v1alpha3, acme.cert-manager.io/v1beta1* APIs on version 1.4.0, we decided to create a compability Matrix between OpenTelemetry Operator and Cert-manager.
+
+| OpenTelemetry Operator | Cert-Manager         |
+|------------------------|----------------------|
+| v0.37.1                | v1.4.0 to v1.6.1     |
+| v0.37.0                | v1.4.0 to v1.5.4     |
+| v0.36.0                | v1.4.0 to v1.5.4     |
+| v0.35.0                | v1.4.0 to v1.5.4     |
+| v0.34.0                | v1.4.0 to v1.5.4     |
+| v0.33.0                | v1.4.0 to v1.5.4     |
+| v0.32.0 (skipped)      | n/a                  |
+| v0.31.0                | v1.4.0 to v1.5.4     |
+| v0.30.0                | v1.4.0 to v1.5.4     |
+| v0.29.0                | v1.4.0 to v1.5.4     |
+| v0.28.0                | v1.4.0 to v1.5.4     |
+| v0.27.0                | v1.4.0 to v1.5.4     |
+
+In pursuit of continuous improvement, a variable named `CERTMANAGER_VERSION` which can be run:
```

Comment:
This is probably more suitable for the contributing guide. End-users of this operator would probably not benefit from this information.